libs/barista-components/tag/src/tag-add/tag-add.ts

```diff
@@ -163,6 +176,12 @@ export class DtTagAdd implements AfterViewInit, OnDestroy {
     this._overlayDir.positionChange.pipe(take(1)).subscribe(() => {
       this._panel.nativeElement.scrollTop = 0;
     });
+    setTimeout(() => {
```

Comment:
So, this **works** well, but results in warning logs, because a div isn't focusable.
Did you have something else in mind?
![image](https://user-images.githubusercontent.com/12587509/118935796-31daad80-b94c-11eb-8e22-46791e4aff45.png)

Reply:
I guess I could make the focus-trap smaller. Then I wouldn't need `cdkFocusInitial`. How important is it that the close button is part of the focus trap?

Reply:
The close button needs to be in the focusTrap imo. You can also get the FocusTrap instance and call focusFirstTabbableElement on it instead of your setTimeout.

Reply:
As discussed, I'll leave it the way it is and add a comment.